06-building-a-program-start-to-finish/writing-tests/output/app-spec.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
```

Comment:
The name of this file (_**app-spec.js**_) is not appropriate. Please confirm with your peers on the preferred naming convention of test files.